mappings/net/minecraft/client/render/entity/EntityRendererStrayOverlay.mapping

```diff
@@ -0,0 +1,10 @@
+CLASS none/bvz net/minecraft/client/render/entity/EntityRendererStrayOverlay
+	FIELD a OVERLAY_TEXTURE Lnone/kp;
```

Comment:
Is that a universal naming scheme for entity textures? In this case the texture is `textures/entity/skeleton/stray_overlay.png`